### DISCUSSION TITLE
Adding a Font

### DISCUSSION THREAD

**Discussion by sazstuff** [chipzoller/hugo-clarity#369] [Q&A]
What is the recommended way to add a font?  I have my own font that is in .woff format and I am not sure where to put all of the good bits for it to work.  I have checked the documentation and did not see anything obvious about adding fonts.

**Comment by rootwork** [comment]
On my personal site, I have my font files in `/static/fonts`. I have a [Sass partial](https://github.com/rootwork/rootwork.org/blob/main/hugo/assets/sass/vars/_fonts.scss) that uses code generated by [Google Webfonts Helper](https://google-webfonts-helper.herokuapp.com/fonts) to reference the font files, and then at the bottom sets the CSS variables. (I have my Sass partials imported separately to `_override.scss`, but you could just put all of this directly in that file, too.) Finally, I have the [`customFonts` option in config](https://github.com/chipzoller/hugo-clarity/#global-parameters) set to `true` to prevent default font files from loading.

You might be able to do some of that using the [`customCSS` option](https://github.com/chipzoller/hugo-clarity/#custom-css-and-js) too, if you wanted.

I think we would probably be happy to accept PRs to make this kind of customization easier, or clearer in the docs!

**Comment by sazstuff** [comment]
Thanks!  I will give that a go and see what happens!  I really appreciate
the quick response and I think the theme is excellent!
